templates/shared/contextual_footers/_openstack_install_documentation.html

```diff
@@ -0,0 +1,21 @@
+<div class="col-4 p-divider__block">
+    <h3 class="p-heading--four">Documentation</h3>
+
+    <ul class="p-list">
+        <li class="p-list__item"><a
+            href="https://microstack.run/docs"
+            class="p-link--external"
+            onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Contextual footer link', 'eventAction' : 'openstack documentation', 'eventLabel' : 'MicroStack', 'eventValue' : undefined });">MicroStack documentation</a>
+        </li>
+        <li class="p-list__item"><a
+            href="https://jaas.ai/openstack-base/bundle/73"
+            class="p-link--external"
+            onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Contextual footer link', 'eventAction' : 'openstack documentation', 'eventLabel' : 'OpenStack Charms documentation', 'eventValue' : undefined });">Charmed OpenStack documentation</a>
+        </li>
+        <li class="p-list__item"><a
+            href="https://docs.openstack.org/charm-guide/latest/"
+            class="p-link--external"
+            onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Contextual footer link', 'eventAction' : 'openstack documentation', 'eventLabel' : 'Charmed OpenStack', 'eventValue' : undefined });">Upstream OpenStack Charms documentation</a>
+        </li>
+    </ul>
```

Comment:
Not sure if these are supposed to rendered as bullet points?

Reply:
Well, the standard design e.g. https://ubuntu.com/openstack does not use bullet points. I am reusing the existent styling.